client/components/chart/index.jsx

```diff
@@ -12,14 +12,18 @@ import { noop, throttle } from 'lodash';
  */
 import BarContainer from './bar-container';
 import { hasTouch } from 'lib/touch-detect';
+import Tooltip from 'components/tooltip';
 
-class Chart extends React.Component {
+class Chart extends React.PureComponent {
 	state = {
 		data: [],
 		isEmptyChart: false,
 		maxBars: 100, // arbitrarily high number. This will be calculated by resize method
 		width: 650,
 		yMax: 0,
+		tooltipContext: null,
+		tooltipPosition: 'bottom left',
```

Comment:
Is this default value ever needed? `setTooltip` always sets it to some other value, and `setTooltip( null )` resets it to `undefined`.

Reply:
It's probably not but isn't it a good practice to define default state for a component?

Reply:
In this case, you never use the default anyway -- so it could be `null` as well.